packages/botonic-react/src/webchat/devices/scrollbar-controller.js

```diff
@@ -51,8 +51,10 @@ export class ScrollbarController {
 
   hasScrollbar() {
     const scrollableArea = getScrollableArea(this.webchat)
+
     const isScrollable =
-      scrollableArea.visible.clientHeight - scrollableArea.full.clientHeight < 0
+      scrollableArea.full.scrollHeight > scrollableArea.visible.clientHeight
```

Comment:
I understand that it has to be the same result but why didn't you keep the same operation?

Reply:
i removed full and visible properties form object, now they are get directly from `ScrollableMessagesListRef` and `ChatAreaRef`